setup.py

```diff
@@ -50,7 +50,7 @@
         'google-api-python-client==1.6.4',
         'google-auth==1.2.1',
         'google-auth-httplib2==0.0.3',
-        'azure',
+        'azure==2.0.0',
```

Comment:
it is the old version, change to 3.0.0 please

Reply:
3.0.0 doesnot work properly , pls check by yourself